docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -211,7 +226,7 @@ supports Container Network Interface (CNI) based networks (and does not support
 
 Several projects provide Kubernetes pod networks using CNI, some of which also
 support [Network Policy](/docs/concepts/services-networking/networkpolicies/). See the [add-ons
-page](/docs/concepts/cluster-administration/addons/) for a complete list of available network add-ons.
+page](/docs/concepts/cluster-administration/addons/) for a complete list of available network add-ons. IPV6 support was added in CNI v0.6, so for IPV6 based network please use CNI version 0.6 or higher.
```

Comment:
sub IPV6 with IPv6

sub 0.6 with 0.6.0 and link to the cni 0.6.0 release page: https://github.com/containernetworking/cni/releases/tag/v0.6.0

Please note that CNI bridge and local-ipam are the only supported drivers in 1.9. Link to these drivers:
https://github.com/containernetworking/plugins/blob/master/plugins/main/bridge/README.md
https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/README.md